addons/hr_attendance/tests/test_attendance_process.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+
+import time
+
+from openerp.exceptions import ValidationError
+from openerp.tests import common
+
+
+class TestAttendanceProcess(common.TransactionCase):
+
+    def setUp(self):
+        super(TestAttendanceProcess, self).setUp()
+
+        self.ResUsers = self.env['res.users']
+        self.ResEmployee = self.env['hr.employee']
+        self.Attendance = self.env['hr.attendance']
+        self.hr_employee_niv = self.ref('hr.employee_niv')
+        # Create a user as 'HR Attendance Officer.'
+        self.res_users_attendance_officer = self.ResUsers.create({
```

Comment:
It looks me part of the test not setup, may be you can put this in test_hr_attendance_process